jobs/resources-import-normalized-consumer/index.ts

```diff
@@ -0,0 +1,138 @@
+/**
+ * Copyright (C) 2021-2023 Technology Matters
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see https://www.gnu.org/licenses/.
+ */
+
+import { ResourceImportProcessorError } from '@tech-matters/hrm-job-errors';
+import fetch from 'node-fetch';
+// import { getSsmParameter } from '@tech-matters/hrm-ssm-cache';
+// import { SQS } from 'aws-sdk';
+// eslint-disable-next-line prettier/prettier
+import type { SQSBatchResponse, SQSEvent, SQSRecord } from 'aws-lambda';
+// eslint-disable-next-line prettier/prettier
+import type { ImportRequestBody } from '@tech-matters/hrm-types';
+
+
+const hrmResourcesApiUrl = process.env.hrm_resources_api_url as string;
+const hrmEnv = process.env.NODE_ENV;
+
+const postNormalizedResourcesBody = async (apiKey: string, message: ImportRequestBody) => {
+    const url = `${hrmResourcesApiUrl}/import`;
+
+    const options = {
+      method: 'POST',
+      headers: {
+        Accept: 'application/json',
+        'Content-Type': 'application/json',
+        // 'Authentication': `Basic ${apiKey}`,
+      },
+      body: JSON.stringify(message),
+    };
+    const response = await fetch(url, options);
+    return response;
+};
+
+const processRecord = async (message: ImportRequestBody): Promise<void> => {
```

Comment:
I think we can name this something a bit less generic, like `writeRecord` pr `upsertRecord`